central/integrationhealth/datastore/datastore_impl.go

```diff
@@ -12,9 +12,9 @@ import (
 )
 
 var (
-	imageSAC    = sac.ForResource(resources.ImageIntegration)
-	notifierSAC = sac.ForResource(resources.Notifier)
-	backupSAC   = sac.ForResource(resources.BackupPlugins)
+	imageSAC    = sac.ForResource(resources.Integration)
+	notifierSAC = sac.ForResource(resources.Integration)
+	backupSAC   = sac.ForResource(resources.Integration)
```

Comment:
I think we can now have a single var and simplify `readAllowed()` and `writeAllowed()`.